src/goto-instrument/dot.cpp

```diff
@@ -41,7 +41,7 @@ class dott
 
   unsigned subgraphscount;
 
-  std::list<exprt> function_calls;
+  std::list<binary_exprt> function_calls;
```

Comment:
Since this is specifically containing function calls, could this be a more specific type than `binary_exprt`?

Reply:
I've made it a pair.